lib/informativeTests/shared/testURL.js

```diff
@@ -1,13 +1,27 @@
+import { createRequire } from 'module'
 import { request } from 'undici'
 
+/**
+ * @type {{
+ *  name: string
+ *  version: string
+ * }}
+ */
+const packageInfo = createRequire(import.meta.url)('../../../package.json')
+
 /**
  * @param {string} url
  * @param {() => void} onError
  */
 export default async function testURL(url, onError) {
+  // set user-agent to csaf-validator-lib/VERSION
+  const userAgent = `${packageInfo.name.split('/').at(-1)}/${packageInfo.version}`
   try {
     const res = await request(url, {
       method: 'HEAD',
+      headers: {
+        'user-agent': userAgent
```

Comment:
According to the [documentation](https://www.rfc-editor.org/rfc/rfc2616#section-14.43) I checked, `User-Agent` is the correct spelling.
```suggestion
        'User-Agent': userAgent